src/dropdown/src/styles/index.cssr.ts

```diff
@@ -64,6 +66,16 @@ export default cB('dropdown-menu', `
           color: 'var(--option-text-color-active)'
         })
       ]),
+      cM('disabled', {
+        color: 'var(--text-color-disabled)',
+        cursor: 'not-allowed',
+        opacity: '.45',
+        backgroundColor: 'var(--color-disabled)'
+      }, [
```

Comment:
Use template string to get hight perf.
eg,
```
cM('disabled', `
  ...
`, [...
```

Reply:
don't understand
I see other docs  are like this